### ISSUE TITLE
specifying a default value of null fails to show the palette

### ISSUE THREAD

**Issue by sztanpet** [Simonwep/pickr#131]
#### Do you want to request a **feature** or report a **bug**? 

bug

#### What is the current behavior?

specifying a default value of null fails to show the palette

#### If the current behavior is a bug, please provide the steps to reproduce and create a JSFiddle.

[JSFiddle](https://jsfiddle.net/jdnup9gs/)

#### What is the expected behavior?

show the proper palette right away, instead of having to interact with it first

#### Your environment:
```
Version (see Pickr.version): 1.2.6
Used bundle (es5 or normal one): tried both, can reproduce in either
Used theme (default is classic): monolith
Browser-version:  chrome 75.0.3770.142
Operating-system:  win10
```

**Comment by sztanpet** [comment]
I have received some feedback about my ticket being not as clear as I would like, so sorry for that, let me explain a bit more.

My use-case is giving the user the ability of overriding the color of specific elements. By default the application doesn't actually know the default color of the element (hence the null default) and when clearing the color, it goes back to that null default. Probably the key detail that I missed to mention is that opacity is disabled (not in the original fiddle, here is the updated fiddle [JSFiddle](https://jsfiddle.net/xer98a4d/)).

The expectation is to show the X for the un-opened color-picker, to signify that there is no color set. When the user opens it, I expect the picker to show the color palette right away without any opacity shenanigans, the checkerboard pattern is invalid in this case. The previous color is also wrong, but that might belong in another ticket.

tl;dr: with default: null, the palette shows the checkerboard pattern with 0% opacity until the first click which resets opacity to 100%, disregarding even the opacity: false setting

**Comment by Simonwep** [comment]
Yeah, but what color should be visible? I suggest to use the `init` event to solve that, since if you use `null` as default, which isn't a color, the initialization fails: https://jsfiddle.net/y4g5ahd1/

**Comment by sztanpet** [comment]
what color? no color, thats what null represents right? it should be symmetric with what `clear` returns (a null), what that means in this case is definitely open to interpretation, but for example: the interaction.input should be empty, the previous color should be the transparent grid and the selection circle can default to wherever really but not showing it at all would be preferred

anyway, just an idea, and sure I can work around missing functionality, feel free to close the ticket if that is your decision

**Comment by Simonwep** [comment]
It should be as configurable as possible, and a color **must** be chosen, pickr can't display **no** color :D An empy input would look like an error IMO, as well as if the previous color would be transparent (which **would be again a color lol**) which would also look like an error.
